pkgs/top-level/stage.nix

```diff
@@ -246,6 +246,8 @@ let
             gnueabihf = lib.systems.parse.abis.musleabihf;
             musleabi = lib.systems.parse.abis.musleabi;
             musleabihf = lib.systems.parse.abis.musleabihf;
+            muslabin32 = lib.systems.parse.abis.muslabin32;
+            muslabi64 = lib.systems.parse.abis.muslabi64;
```

Comment:
If you had ` gnuabi64 = muslabi64` and friends, you will fix the issue.

The point of this bit here is to be an ideompotent glibc -> musl function. Having the musl entries ensures that it won't normalize `musl*` to `musl`, but there is still the `gnuXXX` -> `muslXXX` part to worry about! Sorry I forgot about that when I left the suggestion before.

This should fix the `pkgsStatic` issue you identified above.

(BTW, someone should eventually should pull this out put it in `lib/systems`, but that's a project for another PR.)

Reply:
Implemented in latest push, please review.

(I tested evaluation; but my rebuild to verify the fix is still running)